pkg/controller/nexus/event/service.go

```diff
@@ -0,0 +1,66 @@
+//     Copyright 2020 Nexus Operator and/or its authors
+//
+//     This file is part of Nexus Operator.
+//
+//     Nexus Operator is free software: you can redistribute it and/or modify
+//     it under the terms of the GNU General Public License as published by
+//     the Free Software Foundation, either version 3 of the License, or
+//     (at your option) any later version.
+//
+//     Nexus Operator is distributed in the hope that it will be useful,
+//     but WITHOUT ANY WARRANTY; without even the implied warranty of
+//     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//     GNU General Public License for more details.
+//
+//     You should have received a copy of the GNU General Public License
+//     along with Nexus Operator.  If not, see <https://www.gnu.org/licenses/>.
+
+package event
```

Comment:
I think this package could be moved to `pkg/client/kubernetes` instead, since there's no information regarding the controller itself.

Reply:
That's a good point! I think you meant `pkg/cluster/kubernetes` though, right?  My only objection is that keeping this in a separate package promotes re-usability by making it a small, concise package that manages events and only events. Wdyt of `pkg/cluster/event`?

Reply:
I'd like to call this like `kubernetes.RegisterEvent()` and not `event.RegisterEvent()` 😝

Reply:
I'm confused, the functions we expose are `RaiseInfoEventf()` and `RaiseWarnEventf()`. They would be `event.RaiseInfoEventf()` and `event.RaiseWarnEventf()`. A bit repetitive, but I don't think it's very harmful... I think it's a worthy trade-off to keep package cohesion high. Or do you actually want to rename them to `RegisterEvent()`?

Reply:
> A bit repetitive, but I don't think it's very harmful

Event is a Kubernetes API call, that's why I think would make much more sense then `event` package. We don't want to keep creating too many packages.

Reply:
Ok, we'll keep it in `pkg/cluster/kubernetes`